src/structures/GuildMember.js

```diff
@@ -230,6 +230,17 @@ class GuildMember extends Base {
   get displayName() {
     return this.nickname || this.user.username;
   }
+  
+  /**
+   * The display tag of a user, a normal tag if they don't have a nickname
+   * If someone's nickname is hydrabolt, but their username is hydar, displayTag will be hydrabolt#0001
+   * @type {string}
+   * @readonly
+   */
+  
+  get displayTag() {
+    return this.nickname ? this.nickname + this.user.discriminator : this.user.tag;
```

Comment:
`${this.nickname}#${this.user.discriminator}` in a template literal instead, right now it'll be "hydrabolt0001"